doc/changes/latest.inc

```diff
@@ -283,6 +283,8 @@ Bug
 
 - Fix bug in :meth:`mne.preprocessing.ICA.plot_properties` where time series plot doesn't start at the proper tmin by `Teon Brooks`_
 
+- `~mne.Report` now applies baseline correction by default when rendering (whitended) evoked data, as the documentation always suggested; pass ``baseline=None`` to disable this behavior by `Richard Höchenberger`_
```

Comment:
```suggestion
- `~mne.Report` now applies baseline correction by default when rendering (whitened) evoked data, as the documentation always suggested; pass ``baseline=None`` to disable this behavior by `Richard Höchenberger`_
```

I am not sure the report should do anything more than what was done during processing. It may give some impression of the data in the report that is not what will be done eg before source imaging. Report should not do any computation logic. If need be the sample data should be fixed.

Reply:
or report should have a baseline parameter exposed.

Reply:
actually you have one but it should not be (None, 0) by default. So I would revert the change can set it in the tutorial

Reply:
I've added a new example to the tutorial, showing how to use the `baseline` parameter and which effect it has on the data vizualization.

I then also realized that in the existing example for whitening, it's stated that the data is whitened and baseline-corrected; however it was only whitened. I have now added baseline correction there too.